assets/rbac/csi_driver_role.yaml

```diff
@@ -33,3 +33,21 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments/status"]
     verbs: ["patch"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: ["cns.vmware.com"]
+    resources: ["triggercsifullsyncs"]
+    verbs: ["create", "get", "update", "watch", "list"]
+  - apiGroups: ["cns.vmware.com"]
+    resources: ["cnsvspherevolumemigrations"]
+    verbs: ["create", "get", "list", "watch", "update", "delete"]
+  - apiGroups: ["cns.vmware.com"]
+    resources: ["cnsvolumeoperationrequests"]
+    verbs: ["create", "get", "list", "update", "delete"]
+  - apiGroups: [ "snapshot.storage.k8s.io" ]
+    resources: [ "volumesnapshots" ]
+    verbs: [ "get", "list" ]
+  - apiGroups: [ "cns.vmware.com" ]
+    resources: [ "csinodetopologies" ]
```

Comment:
I didn't know these are required? Who is installing these CRDs? Or are we carrying rbac rules so as driver can talk to CNS apis?

Reply:
Some of these CRDs are created by syncer based on the enabled features: https://github.com/bertinatto/vsphere-csi-driver/blob/aaa8a7710fccddc95b32857dd16feea1f391dd0e/pkg/syncer/cnsoperator/manager/init.go#L64

Without these I get permission error in the logs (probably because I enabled CSI migration for the driver).